### ISSUE TITLE
unable to use react-native-markdown-renderer in Expo

### ISSUE THREAD

**Issue by min9nim** [mientjan/react-native-markdown-renderer#80]
in Expo

below error occured

```
The package at "node_modules/markdown-it/lib/index.js" attempted to import the Node standard library module "punycode". It failed because React Native does not include the Node standard library. Read more at https://docs.expo.io/versions/latest/introduction/faq.html#can-i-use-nodejs-packages-with-expo
ERROR

Building JavaScript bundle: error
```

**Comment by Norfeldt** [comment]
I know that this doesn't help you. But I just installed it via `yarn add react-native-markdow-renderer` in expo SDK 30 and it works.

**Comment by cseelus** [comment]
Yep can confirm it works with Expo. We're using `react-native-markdown-renderer` with our Expo apps for a few months now.

**Comment by rauldeheer** [comment]
@cseelus I'm using SDK 31.0.0 and I'm not able to get this working. I'm getting this error:

`Unable to resolve module `punycode` from `-----`: Module `punycode` does not exist in the haste module map or in these directories: ----`

**Comment by cseelus** [comment]
@rauldeheer Never saw that error. Currently I'm using a fork that re-enables the rendering of headings with React Native >= 0.57.4, maybe you can try that:    

```json
"react-native-markdown-renderer": "imeos/react-native-markdown-renderer#react-native-0.57.4"
```

**Comment by jackswiggett** [comment]
I had the same issue. I fixed it by installing `punycode` directly from npm:

```
npm install punycode
```

**Comment by AdamZaczek** [comment]
Works for me

**Comment by mientjan** [comment]
This is resolved in v4.0.0. The library works with modern Expo apps and includes a working Expo example app in the repository (`example/` directory).
